scripts/process_data.py

```diff
@@ -127,7 +103,7 @@ def main(self) -> None:
         if not self.skip_colmap:
             colmap_dir.mkdir(parents=True, exist_ok=True)
 
-            (sfm_tool, feature_type, matcher_type) = find_tool_feature_matcher_combination(
+            (sfm_tool, feature_type, matcher_type) = process_data_utils.find_tool_feature_matcher_combination(
```

Comment:
Can you raise an error if no valid tool is found.

Reply:
find_tool_feature_matcher_combination returns (None,None,None) when no valid tool is found, which brings to the message CONSOLE.log("[bold yellow]Warning: Invalid combination of sfm_tool, feature_type, and matcher_type ").

I'll add a sys.exit(1) after the message is printed.

Reply:
I changed the message to "[bold red]Invalid combination of sfm_tool, feature_type, and matcher_type, exiting" and added sys.exit(1)